src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -60,10 +79,11 @@ const SearchResult = ({ form }) => {
 
       <dd className="vads-u-padding-bottom--3">
         <a
-          download={form.attributes.url}
+          download={deriveDownloadAttr(form)}
           href={form.attributes.url}
           onClick={onDownloadClick(form.attributes.url)}
```

Comment:
The value part of `download={value}` is just an _optional_ file name. If it isn't passed, the link will still try to force a download (PDF or not). The only difference is that without specifying a value, the browser will decide the name of the saved file. So, I don't think we need to specific a value for `download`. I thought we did before, but that was only because we were passing the entire `form.attributes.url` as the value, creating a weird file name. I think with a plain `download` attribute, the browser will use the last part of the URL as the download name, which makes sense to me.

I think we could refactor this to make it clearer what circumstances lead to attempting a download via JavaScript.  Example -

```jsx
const SearchResult = ({ form }) => {
  // ...
  const linkProps = {};
  const isCrossDomain = new URL(form.attributes.url).hostname !== window.location.hostname;
  const isInternetExplorer = !!navigator.msSaveBlob;
  const isPDF = !!pdf;

  if (isCrossDomain || !isPDF) {
    // Just open in a new tab if we'd otherwise hit a CORS issue or if the form URL isn't a PDF
   linkProps.target = '_blank';
  } else if (isInternetExplorer) {
    // IE doesn't support the HTML download attribute.
    linkProps.onClick = onDownloadClick(form.attributes.url);
  } else {
    // Use HTML5
    linkProps.download = true;
  }

  return <a href={form.attributes.url} {...linkProps}>Download VA form {form.id} {pdf}</a>;
};

```